dll/win32/shell32/CDefaultContextMenu.cpp

```diff
@@ -1341,6 +1342,29 @@ CDefaultContextMenu::InvokeRegVerb(
     return S_OK;
 }
 
+HRESULT
+CDefaultContextMenu::_DoInvokeCommandCallback(
+    LPCMINVOKECOMMANDINFOEX lpcmi, WPARAM CmdId)
+{
+    BOOL Unicode = (lpcmi->fMask & CMIC_MASK_UNICODE) && lpcmi->cbSize >= FIELD_OFFSET(CMINVOKECOMMANDINFOEX, ptInvoke);
+    WCHAR lParamBuf[MAX_PATH];
```

Comment:
I know the shell is already riddled with MAX_PATH, but is it a good idea to introduce new cases?

Reply:
I don't disagree but this MAX_PATH buffer is only ever used by ANSI callers and should be very rare.